run_dir/static/js/lanes_ordered.js

```diff
@@ -0,0 +1,284 @@
+                            /* If no keys are defined, just replace the data */
+                            case 0: {
+                                for (let [my_key, value] of Object.entries(data)) {
+                                    this.statistics_data[my_key] = {
+                                        ...this.statistics_data[my_key],
+                                        ...value
+                                    };
+                                }
+                                break;
+                            }
+                            /* If one or more keys are defined, merge the data */
+                            default: {
+                                let obj = this.statistics_data;
+                                for (let i = 1; i <= nr_of_keys; i++) {
+                                    obj = obj[my_arguments[i - 1]];
+                                }
+                                Object.assign(obj, data);
+                                break;
+                            }
+                        };
+                    }
+                })
+                .catch(error => {
+                    console.log(error)
+                })
+        },
+        async setDefaults() {
+            this.statistics_data = {};
+            /* axios.get returns a promise, so await will make sure these are run sequentially */
+            await this.fetchStatistics();
+            await this.fetchStatistics('ongoing');
+            await this.fetchStatistics('pending');
+        },
+    }
+})
+
+const app = Vue.createApp(vLanesOrderedMain)
+
+app.component('v-lanes-ordered', {
+    /* Lanes Ordered */
+    mounted: function() {
+        this.$root.setDefaults();
+    },
+    template:
+        /* Using the html comment tag can enable html syntax highlighting in editors */
+        /*html*/`
+        <h1>Lanes Ordered</h1>
+        <div class="row mb-3">
+            <p>
+                Showing number of lanes ordered per status, sequencing platform and flowcell type of corresponding projects. <br>
+                The data is fetched from statusdb projects database which is mirrored from the LIMS.
+            </p>
+            <h5>Instructions:</h5>
+            <p>
+                <ul>
+                    <li>Click on the plus signs or the category label to expand a category.</li>
+                    <li>Click on the minus sign to collapse a category.</li>                
+                    <li>Click on a project to go to the projects' page.</li>
+                    <li>Click on the label of an expanded category to display it in the graph.</li> 
+                </ul>
+                <strong>Note:</strong> The ongoing category includes lanes already sequenced since the status 'ongoing' refers to the status of the project.
+            </p>
+            <div class="my-3">
+                <button class="btn btn-primary btn-lg" @click="this.$root.setDefaults()">Reset Defaults</button>
```

Comment:
Clicking on Reset Defaults seems to give different results sometimes, though could be that my connection is slow?

Reply:
I've experienced similar issues. I can in fact reproduce it now. I'll have a look!